core/src/main/resources/reference.conf

```diff
@@ -1,3 +1,12 @@
+clone-workspace-file-transfer-monitor-dispatcher {
+  type = Dispatcher
+  executor = "thread-pool-executor"
+  thread-pool-executor {
+    fixed-pool-size = 32
```

Comment:
I tested with a 32 thread + 64 thread pool, both finished a 9000-file clone in ~2 minutes.

Reply:
The 'optimal' number would probably vary considerably depending on the hardware available, overall server loads, disposition of tea leaves, etc, anyway.  32 seems sane.